qiskit_ibm_provider/ibm_backend.py

```diff
@@ -345,6 +346,7 @@ def run(
         circuits: Union[
             QuantumCircuit, Schedule, List[Union[QuantumCircuit, Schedule]]
         ],
+        program_id: str = "circuit-runner",
```

Comment:
This exposes the runtime API to the user; if `qiskit-ibm-provider` is supposed to be "low level" as compared to `qiskit-ibm-runtime` I don't think it should do this.